src/Application/ContainerFactory.php

```diff
@@ -51,6 +51,18 @@ public function create(string $tempDirectory): Container
             );
         }
 
+        if (isset($configuration['includes'])) {
+            foreach ($configuration['includes'] as $include) {
+                if(pathinfo($include, PATHINFO_EXTENSION) === 'neon') {
+                    $content = Neon::decodeFile(Path::makeAbsolute($include, Path::getDirectory($this->configurationFile)));
+                    $ignoreErrors = $content['parameters']['twigstan']['ignoreErrors'];
+                    $configuration['parameters']['twigstan']['ignoreErrors'][] = $ignoreErrors;
+                }
+            }
+
+            $configuration['parameters']['twigstan']['ignoreErrors'] = array_merge(...$configuration['parameters']['twigstan']['ignoreErrors']);
+        }
```

Comment:
I'm not so sure about this. 

I wonder if this is even needed, given that includes are already supported?

Reply:
I also tried without that, but ignoreError is always empty